package.json

```diff
@@ -47,7 +47,7 @@
         "@types/node": "^13.7.7",
         "@types/resolve": "1.14.0",
         "@typescript-eslint/eslint-plugin": "^2.31.0",
```

Comment:
Plugin version should match the parser, otherwise some rules might handle AST incorrectly. Though if it works okay with current codebase, and there would be some rules with changed configuration format, I think it's fine to keep it that way for now. I could update it to match my [base config](https://github.com/ark120202/eslint-config/blob/master/bases/typescript.js) later